x-pack/apm-server/main.go

```diff
@@ -100,85 +90,19 @@ type processor interface {
 // newProcessors returns a list of processors which will process
 // events in sequential order, prior to the events being published.
 func newProcessors(args beater.ServerParams) ([]namedProcessor, error) {
-	processors := make([]namedProcessor, 0, 5)
-	const txName = "transaction metrics aggregation"
-	args.Logger.Infof("creating %s with config: %+v", txName, args.Config.Aggregation.Transactions)
-	agg, err := txmetrics.NewAggregator(txmetrics.AggregatorConfig{
-		BatchProcessor:                 args.BatchProcessor,
-		MaxTransactionGroups:           args.Config.Aggregation.Transactions.MaxTransactionGroups,
-		MetricsInterval:                metricsInterval,
-		RollUpIntervals:                rollUpMetricsIntervals,
-		MaxTransactionGroupsPerService: int(math.Ceil(0.1 * float64(args.Config.Aggregation.Transactions.MaxTransactionGroups))),
-		MaxServices:                    args.Config.Aggregation.Transactions.MaxServices,
-		HDRHistogramSignificantFigures: args.Config.Aggregation.Transactions.HDRHistogramSignificantFigures,
-	})
+	processors := make([]namedProcessor, 0, 2)
```

Comment:
```suggestion
	var processors []namedProcessor
```

This is not a hot code path, so let's simplify this